refactor tests

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43743 Update aliasing in tensorexpr fuser
* #43742 use types in the IR instead of vmap
* #43636 Add inline call to profiling executor pipeline
* #43635 Use prim::TensorExprGroup interned symbol
* #43634 Update requires grad property
* #43633 Specialize optionals for grad_sum_to_size
* #43632 Add undefined specializations in backward
* **#43631 refactor tests**
* #43630 Refactor pass to class
* #43629 Add API for unexecuted op

I added a new test for just profiler stuff - I don't think the test should go in test_jit.py. Maybe this should just go in test_tensorexpr_fuser, but I'm not really testing tensorexpr stuff either... LMK

Differential Revision: [D23358810](https://our.internmc.facebook.com/intern/diff/D23358810)

Commit:
refactor tests

[ghstack-poisoned]

Commit:
Update on "refactor tests"




[ghstack-poisoned]

ZolotukhinM commented on test/jit/test_profiler.py:
```diff
@@ -0,0 +1,25 @@
+                       "\tpython test/test_jit.py TESTNAME\n\n"
+                       "instead.")
+
+class TestProfiler(JitTestCase):
```
Is it used anywhere?

eellison replied:
it's imported in test_jit.py, test cases are purposefully empty

ZolotukhinM replied:
What's the purpose of empty tests?

eellison replied:
so the next PR that adds tests here has a smaller set of changes

ZolotukhinM reviewed (approved):
Apart from the previous comment, the change looks good!

Commit:
Update on "refactor tests"


I added a new test for just profiler stuff - I don't think the test should go in test_jit.py. Maybe this should just go in test_tensorexpr_fuser, but I'm not really testing tensorexpr stuff either... LMK

Differential Revision: [D23358810](https://our.internmc.facebook.com/intern/diff/D23358810)

[ghstack-poisoned]

Commit:
Update on "refactor tests"


I added a new test for just profiler stuff - I don't think the test should go in test_jit.py. Maybe this should just go in test_tensorexpr_fuser, but I'm not really testing tensorexpr stuff either... LMK

Differential Revision: [D23358810](https://our.internmc.facebook.com/intern/diff/D23358810)

[ghstack-poisoned]
